doc/source/rawio.rst

```diff
@@ -281,6 +281,32 @@ Read event timestamps and times
     In [42]: print(ev_times)
     [ 0.0317]
 
+Signal streams and signal buffers
+---------------------------------
+
+For reading analog signal **rawio** has 2 important concepts:
```

Comment:
```suggestion
For reading analog signals **neo.rawio** has 2 important concepts:
```

I think we want to say neo rawio here no? The signals needs to be plural either way.